aztec/src/main/kotlin/org/wordpress/aztec/toolbar/RippleToggleButton.kt

```diff
@@ -75,6 +77,11 @@ class RippleToggleButton : ToggleButton {
         }
     }
 
+    override fun onLongClick(view: View?): Boolean {
+        Toast.makeText(context, contentDescription, Toast.LENGTH_SHORT).show()
```

Comment:
We should probably add empty string check for contentDescription. It wont cause NPE, but the toast will be empty when content description is not set.